route/route.go

```diff
@@ -11,6 +12,8 @@ import (
 	gometrics "github.com/rcrowley/go-metrics"
 )
 
+var StripPrefixRegex *regexp.Regexp = regexp.MustCompile("modifier-/strip-prefix")
```

Comment:
Would love to hear thoughts on naming here. I'm not overly opinionated, I also considered `urlmodifier-/strip-prefix` and `modifier-/strip`